### PR TITLE
Create new type definitions for rr-notifications

### PR DESCRIPTION
**Pull request by RobbieGM**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

Additional notes:
For the constraint that says that `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true` -- I do not have these properties explicitly set but I have `strict: true` in `tsconfig.json` so they are implicitly set to true. Hopefully maintainers won't find that this is a problem.

### CODE REVIEW

File: types/rr-notifications/index.d.ts
Comment (RyanCavanaugh) [inline types/rr-notifications/index.d.ts:3]: Should be your username here

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]

File: types/rr-notifications/index.d.ts
Comment (RobbieGM) [reply types/rr-notifications/index.d.ts]: Got it, changed `DefinitelyTyped` to `RobbieGM`. I assume you meant to change that rather than changing `Robbie Moore` to `RobbieGM` right?

File: types/rr-notifications/index.d.ts
Comment (RobbieGM) [inline types/rr-notifications/index.d.ts:3]: I corrected the "definitions by" field here. Could maintainers please take a look at this and approve the PR if it's ready?

**Comment by RobbieGM** [comment]
@RyanCavanaugh Could you review the new changes please? Sorry I don't know how to "awaiting reviewer feedback" project board.

**Review by andrewbranch — APPROVED** [review]

**Pull request merged**
